supplementary_style_guide/glossary_terms_conventions/product_conventions/openshift-container-storage.adoc

```diff
@@ -0,0 +1,147 @@
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-bucket]]
+==== namespace bucket (noun)
+*Description*: Namespace buckets enable connecting data repositories on different providers together so that the interaction with data is possible through a single unified view. Data written to namespace buckets are kept plain and not deduped, compressed, or encrypted. The data can be consumed directly from the repository.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[namespace-store]]
+==== namespace store (noun)
+*Description*: A type of a resource for Multicloud Object Gateway that is used to store plain data and is used by namespace buckets. The supported stores are S3, Azure, and S3 compatible.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket]]
+==== object bucket (noun)
+*Description*: A container for storing objects.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[object-bucket-claim]]
+==== object bucket claim (noun)
+*Description*: Creates a new bucket and an application account in Multicloud Object Gateway or RADOS Gateway (RGW) with permissions to the bucket, including a new access key and secret access key. An object bucket claim can be used to request an S3 compatible bucket for the workloads.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[persistent-volume]]
+==== persistent volume (noun)
+*Description*: A persistent volume (PV) is a piece of storage in the cluster that has been provisioned by an administrator or dynamically provisioned using storage classes. 
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[persistent-volume-claim]]
+==== persistent volume claim (noun)
+*Description*: A persistent volume claim (PVC) is a request for storage by a user.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[red-hat-openshift-container-storage]]
+==== Red Hat OpenShift Container Storage (noun)
+*Description*: Red Hat software-defined storage for containers that helps to develop and deploy applications quickly and efficiently across cloud platforms.
+
+*Use it*: yes
+
+*Incorrect forms*: OCS
+
+*See also*: 
+
+[discrete]
+[[rook]]
+==== Rook (noun)
+*Description*: Rook is an orchestrator for multiple storage solutions, each with a specialized Kubernetes Operator to automate management.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[rook-ceph-operator]]
+==== Rook-Ceph Operator (noun)
+
+*Description*: The Rook-Ceph Operator automates the packaging, deployment, management, upgrading, and scaling of persistent storage and file, block, and object services.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+[discrete]
+[[storage-class]]
+==== storage class (noun)
+*Description*: A storage class provides a way to describe the classes of storage offered. OpenShift Container Storage offers block, shared file system, and object classes.
```

Comment:
Passive > active suggestion:

`Use storage classes to describe the types of storage a product offers. OpenShift Container Storage offers block, shared file system, and object classes.`

Also, just wondering if 'OpenShift Container Storage' is talking really about 'OpenShift Container Platform' ? (only because I couldn't find 'OpenShift Container Storage' as a product in the **Official Red Hat product and solution names list**). Ignore if I'm wrong :)

Reply:
Done. 
Yes, OpenShift Container Storage was not listed in the Glossary.